auraed/Cargo.toml

```diff
@@ -33,42 +33,34 @@ name = "auraed"
 version = "0.1.0"
 edition = "2021"
 authors = ["The Aurae Authors", "Kris Nóva <kris@nivenly.com>"]
+license = "Apache-2.0"
 
 [[bin]]
 name = "auraed"
 path = "src/bin/main.rs"
 
 [dependencies]
+anyhow = {workspace = true}
+aurae-proto = { workspace = true }
+cgroups-rs = { git = "https://github.com/kata-containers/cgroups-rs", branch = "main"}
```

Comment:
Wondering why we aren't using the crate from crates.io

Reply:
we are.. i haven't removed this.